context/AuthHelper.tsx

```diff
@@ -49,3 +55,29 @@ export const loginAccount = async ({
     return error as Error;
   }
 };
+
+/**
+ * Log out and clear session state
+ * @returns {Promise<void | Error>}
+ */
+export const logoutFunction = async ({
+  setIsSignedIn,
+  router,
+  resetUser,
+}: logoutType) => {
+  try {
+    await account.deleteSession('current');
+    setIsSignedIn(false);
+    resetUser(); // Reset user data in the store
+    toast.custom(
+      <Alert variant={AlertVariants.Success} message="Logged Out" />,
+    );
+    router.push('/login');
+  } catch (error) {
+    toast.custom(
+      <Alert variant={AlertVariants.Error} message="Logout failed!" />,
+    );
+    console.error('Logout error:', error);
+    return error as Error;
```

Comment:
Does this trigger the error? By returning the error, I don't believe the catch in AuthContextProvider will throw a true error.

Reply:
should I delete the try catch in AuthContextProvider? is there too many try catches?

Reply:
No. What I'm saying is will this throw and error if we're just returning the error object?

Reply:
the test for logging out successfully works, but I did take this out as it's not needed.